code/modules/sanity/breakdowns.dm

```diff
@@ -432,6 +435,93 @@
 		holder.owner.stats.removeTempStat(stat, "Obsession")
 	..()
 
+
+/datum/breakdown/common/nostalgia
+	name = "Nostalgia"
+	restore_sanity_post = 40
+	var/area/nostalgic_target
+
+
+	start_messages = list(
+		"You feel that you've left something somewhere aboard. Something important...",
+		"There is a place where I understood my true self. I should get there again.",
+		"You suddenly feel that some place is pulling you towards itself slowly yet powerfully..."
+	)
+	end_messages = list(
+		"Finally, I am here."
+	)
+
+
+
+/datum/breakdown/common/nostalgia/New()
+	..()
+	nostalgic_target = pick(ship_areas)
+
+/datum/breakdown/common/nostalgia/update()
+	. = ..()
+	if (get_area(holder.owner) == nostalgic_target)
+		conclude()
+
+
+/datum/breakdown/common/high_as_hell
+	name = "High as Hell"
+	restore_sanity_post = 100
+
+	start_messages = list(
```

Comment:
Looks like some copypaste from nostalgia. Make them related to getting high as hell.

Reply:
Oof, my bad, it was a placeholder.